nsx-PowerOps.ps1

```diff
@@ -818,6 +829,425 @@ function parseSSHOutput ($fileToParse, $findElements, $direction) {
+    $summaryTitleRange.merge() | Out-Null
+
+    $summaryHeaderRange = $wsSummary.Range($wsSummary.cells.item($summaryRowHeader,$summaryColHeaderEdgeName), $wsSummary.cells.item($summaryRowHeader,$wsSummary.usedRange.columns.count))
+    $summaryHeaderRange.Interior.ColorIndex = $subTitleInteriorColor
+    $summaryHeaderRange.font.bold = $True
+
+    $row = $summaryRowDetail
+    foreach ($item in $summaryEdgeStats.keys) {
+        $wsSummary.Cells.Item($row,$summaryColHeaderEdgeName) = $item
+        $wsSummary.HyperLinks.Add($wsSummary.cells.item($row,$summaryColHeaderEdgeName),"","'$($item)'!A1") | out-null
+        $wsSummary.Cells.Item($row,$summaryColHeaderEdgeId) = $summaryEdgeStats.item($item).id
+        $wsSummary.Cells.Item($row,$summaryColHeaderVipTotal) = $summaryEdgeStats.item($item).vips
+        $wsSummary.Cells.Item($row,$summaryColHeaderVipOpen) = $summaryEdgeStats.item($item).vipsopen
+        $wsSummary.Cells.Item($row,$summaryColHeaderVipClosed) = $summaryEdgeStats.item($item).vipsclosed
+        if ($summaryEdgeStats.item($item).vipsclosed -ge 1) {
+            $wsSummary.Cells.Item($row,$summaryColHeaderVipClosed).Interior.ColorIndex = $fontColorBad
+        }
+        $wsSummary.Cells.Item($row,$summaryColHeaderPoolTotal) = $summaryEdgeStats.item($item).pools
+        $wsSummary.Cells.Item($row,$summaryColHeaderPoolUp) = $summaryEdgeStats.item($item).poolsup
+        $wsSummary.Cells.Item($row,$summaryColHeaderPoolDown) = $summaryEdgeStats.item($item).poolsdown
+        if ($summaryEdgeStats.item($item).poolsdown -ge 1) {
+            $wsSummary.Cells.Item($row,$summaryColHeaderPoolDown).Interior.ColorIndex = $fontColorBad
+        }
+        $wsSummary.Cells.Item($row,$summaryColHeaderMemberTotal) = $summaryEdgeStats.item($item).members
+        $wsSummary.Cells.Item($row,$summaryColHeaderMemberUp) = $summaryEdgeStats.item($item).membersUp
+        $wsSummary.Cells.Item($row,$summaryColHeaderMemberDown) = $summaryEdgeStats.item($item).membersDown
+        if ($summaryEdgeStats.item($item).membersDown -ge 1) {
+            $wsSummary.Cells.Item($row,$summaryColHeaderMemberDown).Interior.ColorIndex = $fontColorBad
+        }
+        $wsSummary.Cells.Item($row,$summaryColHeaderMemberUnknown) = $summaryEdgeStats.item($item).membersUnknown
+        if ($summaryEdgeStats.item($item).membersUnknown -ge 1) {
+            $wsSummary.Cells.Item($row,$summaryColHeaderMemberUnknown).Interior.ColorIndex = $fontColorUnknownState
+        }
+        $row ++
+    }
+
+    $wsSummaryRange = $wsSummary.usedRange
+    $wsSummaryRange.EntireColumn.Autofit() | out-null
+    $wsSummaryRange.Borders.linestyle = $borderLineStyle
+    $wsSummaryRange.Borders.Weight = $borderWeight
+    $wsSummaryRange.HorizontalAlignment = $horizontalAlignmentCenter
+
+    $global:newExcel.worksheets.item("Sheet1").Delete()
+    $global:newExcel.ActiveWorkbook.SaveAs($DocumentPath)
+    $global:newExcel.Workbooks.Close()
+    $global:newExcel.Quit()
```

Comment:
Take a look in the other Excel functions following my merge - you need to call release object for any COM bound object created within the same scope.  In my testing, that is the WB and the excel application object.  you need to check that excel.exe process is destroyed when your function ends.  You will probably find that with the current code, it leaves orphaned processes?

Reply:
ugh ignore my stupid comment - I missed this two lines before!?  How I dont know...?